operator/src/main/java/org/bf2/operator/managers/MetricsManager.java

```diff
@@ -76,16 +79,35 @@ public void onUpdate(Kafka oldObj, Kafka newObj) {
 
     @Override
     public void onDelete(Kafka obj, boolean deletedFinalStateUnknown) {
-        Search.in(meterRegistry).tags(buildKafkaInstanceTags(obj)).meters().forEach(meterRegistry::remove);
-        kafkaMap.remove(Cache.metaNamespaceKeyFunc(obj));
+        deleteMetrics(obj);
     }
 
-    private void createOrUpdateMetrics(Kafka kafka) {
-        AtomicReference<Kafka> ref = kafkaMap.computeIfAbsent(Cache.metaNamespaceKeyFunc(kafka), (k) -> new AtomicReference<>(kafka));
+    static <T> AtomicReference<T> getMetricHolder(T object, Map<String, AtomicReference<T>> cache) {
+        String cacheKey = Cache.metaNamespaceKeyFunc(object);
+        AtomicReference<T> ref = cache.computeIfAbsent(cacheKey, k -> new AtomicReference<T>(object));
         // Note that meterRegistry.gauge() is a no-op if the gauge already exists.  To handle the update case we
         // update the existing atomic reference that is already registered with the gauge to point at the new kafka object.
-        ref.set(kafka);
+        ref.set(object);
+        return ref;
+    }
+
+    public void createOrUpdateMetrics(ManagedKafka managedKafka) {
+        AtomicReference<ManagedKafka> ref = getMetricHolder(managedKafka, managedKafkaMap);
+        Tags tags = MetricsManager.buildKafkaInstanceTags(managedKafka);
 
+        meterRegistry.gauge(MetricsManager.KAFKA_INSTANCE_SUSPENDED, tags, ref, this::suspended);
+        meterRegistry.gauge(MetricsManager.KAFKA_INSTANCE_PAUSED, tags, ref, this::paused);
```

Comment:
@k-wall , for the new gauge indicating the suspended state of an instance (0 or 1), you had requested the metric always be present. This PR also makes the recently-added metric for the paused state follow the same pattern - always present and a value of 0 or 1. Please let me know if you have any concerns with that approach.

Reply:
@tinaselenge FYI

Reply:
@MikeEdgar I think that sounds good